forge-gui/res/cardsfolder/upcoming/evolved_spinoderm.txt

```diff
@@ -0,0 +1,13 @@
+Name:Evolved Spinoderm
+ManaCost:2 G G
+Types:Creature Phyrexian Beast
+PT:5/5
+K:etbCounter:OIL:4
+S:Mode$ Continuous | Affected$ Creature.Self+counters_LE2_OIL | AddKeyword$ Trample | Description$ CARDNAME has trample as long as it has two or fewer oil counters on it. Otherwise, it has hexproof.
+S:Mode$ Continuous | Affected$ Creature.Self+counters_GT2_OIL | AddKeyword$ Hexproof
+S:Mode$ Continuous | Affected$ Creature.Self | AddPower$ -2 | AddToughness$ -1
```

Comment:
i don't think this line is correct

Reply:
Whoops, yeah artifact from a similar card